example/app/layout.tsx

```diff
@@ -1,9 +1,8 @@
-'use client'
```

Comment:
Issue: update examples in readme (remove `'use client'`)

Reply:
done